ecomap/ecomap/Model/EcomapFetcher.h

```diff
@@ -0,0 +1,74 @@
+//
+//  EcomapFetcher.h
+//  EcomapFetcher
+//
+//  Created by Vasilii Kotsiuba on 2/3/15.
+//  Copyright (c) 2015 Vasyl Kotsiuba. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "EcomapURLFetcher.h"
+
+@class EcomapProblemDetails;
+@class EcomapLoggedUser;
+@class EcomapProblem;
+@class EcomapCommentsChild;
+
+@interface EcomapFetcher : NSObject
```

Comment:
I've stopped review of this because it's too big. It should not be. You should separate a lot of logic and make this one more clear.